packages/ui/src/components/va-select/VaSelect.demo.vue

```diff
@@ -11,6 +11,107 @@
       />
       <p>Value: {{ defaultSingleSelect.value }}</p>
     </VbCard>
+    <VbCard
+      title="Autocomplete"
+      style="width: 400px;"
+    >
+      <va-select
+        v-model="defaultSingleSelect.value"
+        class="mb-6"
+        placeholder="Start to write..."
+        :options="defaultSingleSelect.options"
+        autocomplete
+        highlight-matched-text
+      />
+      <p>Value: {{ defaultSingleSelect.value }}</p>
+    </VbCard>
+    <VbCard
+      title="Multiple autocomplete"
+      style="width: 400px;"
+    >
+      <va-select
+        v-model="defaultMultiSelect.value"
+        class="mb-6"
+        placeholder="Start to write..."
+        :options="defaultMultiSelect.options"
+        multiple
+        autocomplete
+        highlight-matched-text
+      />
+      <p>Value: {{ defaultMultiSelect.value }}</p>
+    </VbCard>
+    <VbCard
+      title="Multiple autocomplete with max visible options"
+      style="width: 400px;"
+    >
+      <va-select
+        v-model="defaultMultiSelect.value"
+        class="mb-6"
+        placeholder="Start to write..."
+        :options="defaultMultiSelect.options"
+        :max-visible-options="2"
+        multiple
```

Comment:
Not related to autocomplete, but can't we make "Items not found" formatting a bit better (should occupy the same space as list item)

![image](https://user-images.githubusercontent.com/5394573/217970320-9a330594-ef84-41c4-83e5-17f78e3ccbb7.png)